### DISCUSSION TITLE
How to select multiple lines

### DISCUSSION THREAD

**Discussion by tewe** [extrawurst/gitui#956] [Q&A] [answered]
When working on the diff, I can "reset lines", which asks if I "want to discard 1 selected line". So I assume more than one line at a time is supported. But I'm afraid I can't figure out how to select lines.

Shift+arrow keys does not work in Apple Terminal.

**Comment by extrawurst** [comment, marked as answer]
see https://github.com/extrawurst/gitui/issues/959#issuecomment-950150863
